content/en/blog/2025/announcing-etcd-3.6.md

```diff
@@ -0,0 +1,403 @@
+do not depend on an external raft module. Starting from etcd v3.6, raft was moved to a separate repository ([raft][]),
+and the first standalone raft release is v3.6.0. As a result, etcd v3.6.0 depends on raft v3.6.0.
+
+Please see the table below for a summary:
+
+| etcd versions | bbolt versions | raft versions |
+|---------------|----------------|---------------|
+| 3.4.x         | v1.3.x         | N/A           |
+| 3.5.x         | v1.3.x         | N/A           |
+| 3.6.x         | v1.4.x         | v3.6.x        |
+
+### grpc-gateway@v2
+
+We upgraded [grpc-gateway][] from v1 to v2 via [PR/16595][] in etcd v3.6.0. This is a major step toward
+migrating to [protobuf-go][], the second major version of the Go protocol buffer API implementation.
+
+grpc-gateway@v2 is designed to work with [protobuf-go][]. However, etcd v3.6 still depends on the deprecated
+[gogo/protobuf][] which is actually protocol buffer v1 implementation. To resolve this incompatibility,
+we applied a [patch][] to the generated *.pb.gw.go files to convert v1 messages to v2 messages.
+
+### grpc-ecosystem/go-grpc-middleware/providers/prometheus
+
+We switched from the deprecated (and also archived) [grpc-ecosystem/go-grpc-prometheus][] to
+[grpc-ecosystem/go-grpc-middleware/providers/prometheus][] via [PR/19195][]. This change ensures continued
+support and access to the latest features and improvements in the gRPC Prometheus integration.
+
+## Community
+
+There are exciting developments in the etcd community that reflect our ongoing commitment
+to strengthening collaboration, improving maintainability, and evolving the project’s governance.
+
+### etcd Becomes a Kubernetes SIG
+
+etcd has officially become a Kubernetes Special Interest Group: SIG-etcd. This change reflects
+etcd’s critical role as the primary datastore for Kubernetes and establishes a more structured
+and transparent home for long-term stewardship and cross-project collaboration. The new SIG
+designation will help streamline decision-making, align roadmaps with Kubernetes needs,
+and attract broader community involvement.
+
+### New Contributors, Maintainers, and Reviewers
+
+We’ve seen increasing engagement from contributors, which has resulted in the addition of two new maintainers:
+
+- @fuweid
+- @jmhbnz
+
+Their continued contributions have been instrumental in driving the project forward.
+
+We also welcome two new reviewers to the project:
+
+- @ivanvc
+- @siyuanfoundation
+
+We appreciate their dedication to code quality and their willingness to take on broader review responsibilities
+within the community.
+
+### Introducing the etcd Operator Working Group
+
+To further advance etcd’s operational excellence, we have formed a new working group: [WG-etcd-operator][].
+The working group is dedicated to enabling automatic and efficient operation of etcd clusters which run in
+Kubernetes environment using an etcd-operator.
+
```

Comment:
Idea: We could mention release process improvements, we now have a lot more confidence in our release processes through much better automation which has drastically reduced how long it takes us to release new versions. 

We also now have dedicated release team roles and release shadows, adopting community building best practices from Kubernetes sig-release etc.

Reply:
added a new section "new release team" ,PTAL